frontend/src/pages/CoursesList/CoursesList.tsx

```diff
@@ -1,10 +1,65 @@
 import React from 'react';
 
 import { AuthorizedLayout } from 'components/Layout';
+import { PageContainer, CourseButton, CourseActions } from './styled';
+import { CourseItem } from 'components/Course';
+
+const INITIAL_COURSES = [
```

Comment:
We should move it to constants folder for this time

Reply:
Removed into constants and put into `CoursesList`  file